network/manager.go

```diff
@@ -115,8 +115,22 @@ func (nm *networkManager) Initialize(config *common.PluginConfig, isRehydrationR
 func (nm *networkManager) Uninitialize() {
 }
 
+//Remove the lock on the key-value store
+func (nm *networkManager) unlockStore() error {
```

Comment:
why do we need separate function for unlock alone?

Reply:
I had to release lock in the restore function at multiple blocks, pretty much before every return().